camel-core/src/main/java/org/apache/camel/impl/verifier/ResultErrorHelper.java

```diff
@@ -63,16 +63,31 @@ private ResultErrorHelper() {
         final Set<String> keys = new HashSet<>(parameters.keySet());
 
         for (OptionsGroup group : groups) {
-            if (keys.containsAll(group.getOptions())) {
+            final Set<String> required = required(group.getOptions());
+
+            final ResultErrorBuilder builder = new ResultErrorBuilder()
+                .code(VerificationError.StandardCode.INCOMPLETE_PARAMETER_GROUP)
```

Comment:
Is it true that it is always `INCOMPLETE_PARAMETER_GROUP` ? Maybe another error code `CONTRADICTING_PARAMETER_GROUP` would make sense here, too ?